docs/migrations.md

```diff
@@ -273,6 +281,33 @@ module.exports = class PostRefactoringTIMESTAMP {
         )
     }
 }
+
+```
+By default, it generates CommonJS JavaScript code with the `o` (alias for `--outputJs`) flag, but you can generate also ESM code with the `esm` flag. This is useful for Javascript projects that use ESM modules:
```

Comment:
nit:
```suggestion
By default, it generates CommonJS JavaScript code with the `o` (alias for `--outputJs`) flag, but you can generate also ESM code with the `esm` flag. This is useful for Javascript projects that use ESM:
```

Reply:
Fixed in 51c1a9aa0d01f9d48b27b3515221b2667f25aa78